src/application/wizard/resolvers.ts

```diff
@@ -99,6 +99,17 @@ const resolvers = (wizard: WizardService, userService: UserService): IResolvers
                 },
             ),
         },
+        supportingUploadProgress: {
+            subscribe: withFilter(
```

Comment:
Would be good to get a quick knowledge share amongst the team tomorrow on how this all works. Code looks good but I'm not completely sure of what I'm looking at :)